docs/your-first-snap.md

```diff
@@ -323,7 +323,8 @@ We'll be happy to help you on the mailing list to build a snappy package of
 anything that you are interested in. Choose a good name for it, and you can
 very easily share it in
 [ubuntu myapps](https://myapps.developer.ubuntu.com/dev/click-apps/?format=snap)
-where you go to share it with other snappy users.
+where you go to share it with other snappy users. You can even use Snapcraft to
+[upload for you](upload-your-snap.md)!
```

Comment:
Nice! Thanks for the great work @ricardokirkner. 
And @kyrofa of course for documenting it.